branding/fixed-strings.json

```diff
@@ -2,5 +2,8 @@
   "BRAND_NAME": "oVirt",
   "LEGAL_INFO": "Released under",
   "LEGAL_INFO_LINK_TEXT": "Apache License 2.0",
-  "LEGAL_INFO_LINK_URL": "https://github.com/oVirt/ovirt-web-ui/blob/master/LICENSE"
+  "LEGAL_INFO_LINK_URL": "https://github.com/oVirt/ovirt-web-ui/blob/master/LICENSE",
+  "GITHUB_ISSUES_TRACKER_URL": "https://github.com/oVirt/ovirt-web-ui/issues",
```

Comment:
I think will be enough leave attributes with name `ISSUES_TRACKER_URL` and `ISSUES_TRACKER_NAME`. And then use it in `About` component.

Reply:
Please, update `fixed-strings.json`, I mean, remove `REDHAT_CUSTOMER_PORTAL_URL` and `GITHUB_ISSUES_TRACKER_URL` and add `ISSUES_TRACKER_URL`, `ISSUES_TRACKER_TEXT`